src/com/videojs/providers/HTTPVideoProvider.as

```diff
@@ -597,10 +600,8 @@ package com.videojs.providers{
 
                 case "NetStream.Seek.Notify":
                     _playbackStarted = true;
-                    _isSeeking = false;
                     _hasEnded = false;
                     _model.broadcastEvent(new VideoPlaybackEvent(VideoPlaybackEvent.ON_STREAM_SEEK_COMPLETE, {info:e.info}));
```

Comment:
That event is internal-only so I didn't think it was necessary to rename/refactor it. Its definitely a little weird to see here, I just didn't want to have to track down all the usages and update them.